cmd/keymasterd/main.go

```diff
@@ -1630,6 +1732,27 @@ func Usage() {
 
 func init() {
 	prometheus.MustRegister(certGenCounter)
+	prometheus.MustRegister(authOperationCounter)
+	prometheus.MustRegister(externalServiceDurationTotal)
+	prometheus.MustRegister(certDurationHistogram)
+	tricorder.RegisterMetric(
+		"keymaster/externalServiceDuration/LDAP",
```

Comment:
The tricorder metric name convention is lowercase with hyphens.

Reply:
Done